.github/workflows/run-validation-oidc.yml

```diff
@@ -592,22 +592,22 @@ jobs:
       - name: Azure Login
         uses: azure/login@v1
         with:
-          client-id: ${{ secrets.TEST_CLIENT_ID }}
-          tenant-id: ${{ secrets.TEST_TENANT_ID }}
-          subscription-id: ${{ secrets.TEST_SUBSCRIPTION_ID }}
+          client-id: ${{ secrets.OIDC_TEST_CLIENT_ID }}
+          tenant-id: ${{ secrets.OIDC_TEST_TENANT_ID }}
+          subscription-id: ${{ secrets.OIDC_TEST_SUBSCRIPTION_ID }}
 
       - name: Execute Azure Container Apps Build and Deploy Action
         uses: ./
         with:
           appSourcePath: '${{ github.workspace }}/oryx/tests/SampleApps/DotNetCore/NetCore6PreviewWebApp'
-          containerAppName: ${{ env.TEST_EXISTING_CONTAINER_APP }}
+          containerAppName: ${{ vars.TEST_EXISTING_CONTAINER_APP }}
```

Comment:
What's the reason for this change? It looks like other tests are still referencing `env` and not `var`

Reply:
This one was added because for some reason the action couldn't read the container app name (shown as empty in debug message). It then got fixed after I changed it from env to vars.